http/src/test/java/com/github/awsjavakit/http/RenewTokenWithCredentialsStoredInSecretsManagerTest.java

```diff
@@ -75,11 +78,11 @@ private void persistSecretsInSecretsManager() {
   }
 
   private void setupAuthAndServiceServer() {
-    var response = new OAuthResponse(accessToken, randomString(), randomString());
```

Comment:
the OAuthResponse was a test class, that is now replaced by the OAuthTokenResponse class